src/main/java/org/apache/commons/io/EndianUtils.java

```diff
@@ -261,7 +275,8 @@ public static float swapFloat(final float value) {
     }
 
     /**
-     * Converts an "int" value between endian systems.
+     * Converts an {@code int} value between endian systems.
```

Comment:
We should be clear that the toggle is between big and little-endian. We'll then avoid the shadow of even thinking about middle-endian madness.

Reply:
Good point. Curious, has any system ever actually used middle-endianess or is it just a theoretical possibility? Anyway, let me update this.